etl/steps/data/garden/war/2023-09-25/peace_diehl.meta.yml

```diff
@@ -39,27 +39,37 @@ tables:
         unit: countries
         processing_level: minor
         description_processing: *description_processing
+        display:
+          numDecimalPlaces: 1
```

Comment:
Decimal places should be 0 in all cases? I think they are all counts of countries?

Reply:
totally! nice catch